client/reader/following/style.scss

```diff
@@ -19,11 +19,11 @@
 
 		a {
 			color: #c9eaf5;
-			border-bottom: 1px #c9eaf5 solid;
```

Comment:
It doesn't really, but we've mostly been using thickness, style, then color for the order.